packages/py-api/py_api/controllers/getting_logs_controller.py

```diff
@@ -0,0 +1,44 @@
+import pathlib
+from os.path import exists
+from traceback import format_exc
+from typing import Final
+
+from fastapi import APIRouter
+from py_api.environment import IS_OFFLINE
+from starlette.responses import FileResponse, JSONResponse

```

Comment:
We usually use the JSONResponse from `fastapi.responses` not starlette

Reply:
Fixed